src/api/telemetry/TelemetryValueFormatter.js

```diff
@@ -81,6 +81,15 @@ define([
                 return printj.sprintf(formatString, baseFormat.call(this, value));
             };
         }
+        if (valueMetadata.format === 'string') {
```

Comment:
Can we follow the number formatter method, instead of just changing the parse method, we have to change format and validate function as well. The validate function should return typeof value === 'string'

Reply:
Done.